db/main/migrate/20221214171030_add_purpose_column_to_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddPurposeColumnToTokens < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tokens, :purpose, :integer, limit: 4
```

Comment:
hi @speedywizard - there needs to be a default (0) , otherwise you won't be able to login

Reply:
Hey @GbArc , good point, thanks!